sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -0,0 +1,484 @@
+            if destination == ".":
+                # juju scp does not allow "." as destination
+                destination = Path("./", Path(source).name)
+            subprocess.run(
+                [
+                    "juju",
+                    "scp",
+                    "--container",
+                    TEMPEST_CONTAINER_NAME,
+                    f"{unit}:{source}",
+                    destination,
+                ],
+            )
+
+    def _configure_preflight_check(self) -> False:
+        """Preflight check for configure command."""
+        enabled_plugins = PluginManager.enabled_plugins(self.client)
+        if "observability" not in enabled_plugins:
+            return False
+        return True
+
+    @click.command()
+    def enable_plugin(self) -> None:
+        """Enable OpenStack Integration Test Suite (tempest)."""
+        super().enable_plugin()
+
+    @click.command()
+    def disable_plugin(self) -> None:
+        """Disable OpenStack Integration Test Suite (tempest)."""
+        super().disable_plugin()
+
+    @click.command()
+    @click.argument("options", nargs=-1)
+    def configure_validation(self, options: Optional[List[str]] = None) -> None:
+        """Configure validation plugin.
+
+        Run without arguments to view available configuration options.
+
+        Run with key=value args to set configuration values.
+        For example: sunbeam configure validation schedule="*/30 * * * *"
+        """
+        if not self._configure_preflight_check():
+            raise click.ClickException(
+                "'observability' plugin is required for configuring validation plugin."
+            )
+
+        if not options:
+            console.print(
+                "Config options available: \n\n"
+                "schedule: set a cron schedule for running periodic tests.  "
+                "Empty disables.\n\n"
+                "Run with key=value args to set configuration values.\n"
+                'For example: sunbeam configure validation schedule="*/30 * * * *"'
+            )
+            return
+
+        config_changes = validated_config_args(parse_config_args(options))
+
+        if config_changes.schedule is not None:
+            jhelper = JujuHelper(self.client, self.snap.paths.user_data)
+            with console.status("Configuring validation plugin ..."):
+                run_sync(
+                    jhelper.set_application_config(
+                        OPENSTACK_MODEL,
+                        TEMPEST_APP_NAME,
+                        config={"schedule": config_changes.schedule},
+                    )
+                )
+                console.print(message)
+
+    @click.command()
+    @click.option(
+        "-s",
+        "--smoke",
+        is_flag=True,
+        default=False,
+        help=(
+            "Run the smoke tests only. Equivalent to --regex=smoke. "
+            "If both --regex and --smoke are provided, --regex is ignored."
+        ),
+    )
```

Comment:
It sounds the opposite ... Since we don't have "profiles" now, the shortcut does made sense right now. I believe we have a task track that we should keep it as it is. We should drop it when we have a more advance feature like "profile"

Reply:
Let's just drop it to reduce future work